tools/stress-cluster/cluster/provision.ps1

```diff
@@ -1,11 +1,12 @@
 param (
-    [string]$Environment = 'test',
+    [string]$Environment = 'dev',
```

Comment:
I think the answer is no, but worth double checking if any changes are necessary to the readme.

Reply:
Yup the answer is no